packages/devtools_app/lib/src/screens/memory/framework/memory_controller.dart

```diff
@@ -105,6 +105,10 @@ class MemoryController extends DisposableController
             return OfflineMemoryData.fromJson(data as Map<String, dynamic>);
           },
           shouldLoad: (data) => true,
+          loadData: (data) async {
+            assert(!_initialized.isCompleted);
+            _initializeData(offlineData: data);
```

Comment:
move the asset into the `_initializeData` method, and then pass `_initializeData` here as a tearoff

Reply:
_initializeData has different signature